webforces/server/mongodbworker.py

```diff
@@ -50,52 +50,55 @@ def _populateIds(self):
     def dropAll(self) -> None:
         (self.db["id"]).drop()
         (self.db["users"]).drop()
-        (self.db["algorithms"]).drop()
+        (self.db["algs"]).drop()
         (self.db["tests"]).drop()
         (self.db["tasks"]).drop()
         logger.debug("Collections were dropped")
 
-    # "users"       - get the last user ID + 1
-    # "tasks"       - get the last task ID + 1
-    # user_id + "u" - get ID of the last binded alg for this user + 1
-    # alg_id + "a"  - get ID of the last added test for this alg + 1
+    # "users"                      - get the last user ID + 1
+    # "tasks"                      - get the last task ID + 1
+    # user_id + "u"                - get ID of the last binded alg for this user + 1
```

Comment:
`bound` instead of `binded` :)
Same for error messages.

Reply:
Fixed, ty.